src/style.re

```diff
@@ -72,36 +85,36 @@ let alignContent = (v) =>
   stringStyle(
     "alignContent",
     switch v {
-    | `flexStart => "flex-start"
-    | `flexEnd => "flex-end"
-    | `center => "center"
-    | `stretch => "stretch"
-    | `spaceAround => "space-around"
-    | `spaceBetween => "space-between"
+    | `FlexStart => "flex-start"
```

Comment:
What about making all of these a proper variant instead of a polymorphic one?
Same goes for 
```
`Row, `Wrap and `Flex
``` 
and others if I've missed some

Reply:
Yes, why not.

Reply:
:+1: